pages/contributing.md

```diff
@@ -1,216 +1,896 @@
 ---
 layout: default
 title: Contributing
```

Comment:
I would prefer "Contributor's Guide" instead of "Contributing". Thoughts?

Reply:
Yeah, I think that is appropriate. I'll make the change!